app/models/katello/concerns/smart_proxy_extensions.rb

```diff
@@ -121,6 +133,15 @@ def alternate_content_sources
         SmartProxy.joins(:smart_proxy_alternate_content_sources).where('katello_smart_proxy_alternate_content_sources.smart_proxy_id' => self.id)
       end
 
+      def registration_fqdn
+        url = self.setting('Registration', 'registration_url').presence || self.url
+        name == "rhel8b.fedora.example.com" ? "loadbalancer.example.com" : URI.parse(url).host
```

Comment:
Since you are grabbing the `.host`, you could opt to call this method `registration_host` instead